src/test_l1_dsFPD.c

```diff
@@ -3080,7 +3080,7 @@ void test_l1_dsFPD_negative_dsFPSetLEDState(void)
  * |02|Set all valid indicators to dsFPD_STATE_ON using dsSetFPState()|eIndicator: [Valid Indicator], state: dsFPD_STATE_ON |dsERR_NONE|Ensure the system is initialized|
  * |03|Call dsFPSetLEDState() with valid value|state: dsFPD_LED_DEVICE_ACTIVE|dsERR_NONE|The LED state should be set successfully|
  * |04|Call dsFPGetLEDState() with a valid parameter|state: dsFPDLedState_t*|dsERR_NONE|API should retrieve the current state successfully|
- * |05|Compare the retrieved LED state with the expected value from the profile file
+ * |05|Compare the retrieved LED state with the with the set value in step 03
```

Comment:
this L1 test case we need to use only dsFPGetLEDState() and it's @pre condition API's only used to test 
please update the test case like that and please check the other test case also

Reply:
Checked and updated wherever applicable

Reply:
is there any requirement to enableFPDIndicators() & disableFPDIndicators() even there is no @pre for that API?

Reply:
No enable/disable for GetLEDState or SetLEDState implemented as it is not a prerequesite.